pkgs/by-name/sa/satty/package.nix

```
@@ -43,6 +45,11 @@ rustPlatform.buildRustPackage rec {
 
   postInstall = ''
     install -Dt $out/share/icons/hicolor/scalable/apps/ assets/satty.svg
+
+    installShellCompletion --cmd satty \
+      --bash <(cat completions/satty.bash) \
+      --fish <(cat completions/satty.fish) \
+      --zsh <(cat completions/_satty)
```

```suggestion
      --bash completions/satty.bash \
      --fish completions/satty.fish \
      --zsh completions/_satty
```

This should work? Stop cat abuse :)

---

Fixed :+1: